hr_employee_calendar_planning/models/hr_employee.py

```diff
@@ -68,20 +69,17 @@ class HrEmployeeCalendar(models.Model):
         ),
     ]
 
-    @api.model
```

Comment:
`@api.model` doesn't disappear. Any way, better to decorate it with `@api.model_create_multi` and take advantage of speed optimization in it.

Reply:
Done, I deleted it by mistake when deleting api multis.